src/locales/es/translations/index.ts

```diff
@@ -0,0 +1,19 @@
+import actions from './actions'
+import dashboard from './dashboard'
+import patient from './patient'
+import patients from './patients'
+import scheduling from './scheduling'
+import states from './states'
+import sex from './sex'
+import labs from './labs'
+
+export default {
```

Comment:
Similar to earlier issue, but if we are talking strictly in terms of being comprehensive, I believe you are missing a couple of translation folders. Specifically, terms such as `bloodType` and `medications` are missing. I would also refer to the full English translations here: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/src/shared/locales/enUs/translations/index.ts.